new/LIT7412QeneLaegwala.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7412QeneLaegwala" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለእጓለ፡ ማውተ፡ ነፍስየ፡ ተሐድጋ፡ ለምንት፡ እግዚአብሔር፡ አቡሃ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-ʾǝgʷāla māwta nafsǝya taḥaddǝgā la-mǝnt ʾƎgziʾabǝḥer ʾAbu-hā
```

Comment:
ʾabu-hā

Reply:
"ʾǝgʷāla māwta" means "orphan". The line means smth like: Why are you abandoning my soul, orphan, (you) God, her father... (ተሐድጋ is somewhat difficult to explain, but I think it is the imperfect form with a suffix, 1st order instead of 6th before laringeal does happen).